src/components/Maps.vue

```diff
@@ -13,7 +13,10 @@
       v-model="isModelCreationDialogVisible"
       @returnObject="passModel"
     />
-    <NewMap v-model="isMapCreationDialogVisible" />
+    <NewMap 
+      v-model="isMapCreationDialogVisible" 
+      @returnObject="passMap"
```

Comment:
Event names are recommended [to be kebab-case](https://vuejs.org/v2/guide/components-custom-events.html#Event-Names).

Reply:
True! Will fix it soon!